### ISSUE TITLE
VO stops indicating leg position after statement, "You are on a slider....."

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/qa#684]
**Test device**
MacBook Air (M1 chip)

**Operating System**
11.4

**Browser**
safari latest

**Problem description**
https://github.com/phetsims/qa/issues/676

With VO activated, I was using the arrow keys to move the leg back and forth. Randomly, VO then states that I'm currently on a slider etc. After this statement, when I continue to move the leg back and forth with the arrow keys, VO only states the number of electrons and no longer mentions foot position. 

**Steps to reproduce**
1. Open the sim and tab into it
2. Start voice over
3. Tab over to the leg and move back and forth using the arrow keys

**Visuals**
Issue doesn't happen until ~50 seconds into the video but the entire video allows you to see the before and after.

https://drive.google.com/file/d/1t1FTtdqE37NANoQkzKU6INZ06wTZ9AxB/view?usp=sharing 

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.1/phet/john-travoltage_all_phet.html
Version: 1.6.0-rc.1 2021-07-22 18:05:38 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15
Language: en-us
Window: 1111x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by Nancy-Salpepi** [comment]
Sorry--I put this in the wrong repo. Closing out
